modules/integrations/vercel/commerce/src/app/data.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {SearchBuilder} from '@/utils/search-builder';
```

Comment:
I believe these "@" aliases are what causing the TS error you mentioned in slack. I would probably remove all aliased code and use relative paths

```
/commerce/src/app/data.ts: TypeError: Cannot read properties of null (reading '0')
     [exec] Occurred while linting commerce/src/app/data.ts:6
```